packages/react/src/components/DetailsList/DetailsRow.base.tsx

```diff
@@ -308,6 +313,16 @@ export class DetailsRowBase extends React.Component<IDetailsRowBaseProps, IDetai
         aria-selected={ariaSelected}
         allowFocusRoot={true}
       >
+        {ariaRowDescription ? (
+          <span
+            key="description"
+            role="presentation"
+            id={this._ariaRowDescriptionId}
+            className={this._classNames.screenReaderOnly}
```

Comment:
actually, sorry, one other suggestion -- both `aria-labelledby` and `aria-describedby` work with `hidden` elements. I'd suggest adding `hidden={true}` here instead of exposing the node itself to screen readers. That way screen reader users won't encounter random non-cell text within the table, and browser and screen reader heuristics shouldn't interfere.